src/test/java/pages/stepDefinitions/LoginStepDefinitions.java

```diff
@@ -0,0 +1,71 @@
+package pages.stepDefinitions;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class LoginStepDefinitions {
+
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @Before
+    public void openBrowser() {
+        WebDriver.chromeDriver().setup();
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+
+    }
+
+    @After
+    public void iCloseTheBrowser() {
+        driver.quit();
+
+    }
+
+    @Given("I open Login Page")
+    public void openBrowser() {
+        driver = new ChromeDriver();
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226833364-ba0c6c92-01b1-4d95-a9cf-c61c6e5a8d1e.png)

HINT

```
    @Before
    public void setupBrowser() {
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));

    }

    @After
    public void iCloseTheBrowser() {
        driver.quit();

    }

    @Given("I open Login Page")
    public void openBrowser() {
       driver.get("https://bbb.testpro.io");
    }

```